README.md

```diff
@@ -1,26 +1,25 @@
 - [Overview](#Overview)
-  - [Purpose & Principles](#Purpose-&-Principles)
+  - [Purpose](#Purpose)
+  - [Principles](#Principles)
   - [Architecture Diagram](#Architecture-Diagram)
 - [Features](#Features)
 - [QuickStart](#QuickStart)
   - [Download](#Download)
   - [Build From Source](#Build-From-Source)
-- [Documentation](#Documentation)
-  - [Agent Configuration](#Agent-Configuration)
-  - [Health Check Endpoint](#Health-Check-Endpoint)
-  - [Readiness Check Endpoint](#Readiness-Check-Endpoint)
-  - [Tracing](#Tracing)
-  - [Metrics](#Metrics)
-    - [Prometheus Metrics Support](#Prometheus-Metrics-Support)
+- [User Manual](#User-Manual)
 - [Development Guide](#Development-Guide)  
 - [Licenses](#Licenses)
 
 
 ## Overview
+An agent component for Java system
 
 ### Purpose
```

Comment:
Why doesn't the document contain a title?  Please refer to README.md of other OSS.